bin/bootstrap_practice_exercise.exs

```diff
@@ -243,38 +240,6 @@ url =
 :inets.start()
 :ssl.start()
 
-# .docs/instructions.md
-{:ok, {_status, _header, description}} =
-  :httpc.request(:get, {url ++ ~c"/description.md", []}, [], [])
-
-Mix.Generator.create_file("exercises/practice/#{exercise}/.docs/instructions.md", description)
```

Comment:
configlet can create these files now, so it's better to let it do it (and also there are `introduction.md` files now that this didn't handle)